crates/router/src/core/fraud_check/flows/sale_flow.rs

```
@@ -65,6 +66,12 @@ impl ConstructFlowSpecificData<frm_api::Sale, FraudCheckSaleData, FraudCheckResp
             request: FraudCheckSaleData {
                 amount: self.payment_attempt.amount,
                 order_details: self.order_details.clone(),
+                currency: self.payment_attempt.currency,
+                email: customer.clone().and_then(|customer_data| {
+                    customer_data
+                        .email
+                        .and_then(|email| Email::try_from(email.into_inner().expose()).ok())
```

Shouldn't we propagate the error here?

---

Yes, updated!